assets/css/style.css

```diff
@@ -321,8 +305,57 @@ a {
 .contact-bar-item svg path {
     fill: rgb(116, 135, 129);
 }
+
+
 @media ( max-width: 768px ) {
-    .contact-bar {
+    .navbar {
+        padding-right: 3px;
+        padding-left: 30px;
+    }
+
+    .navbar-menu {
         flex-direction: column;
     }
+
+    .navbar-btn {
+        margin-bottom:-12px
+    }
+
+    /* intro */
+    #intro-section {
+        padding: 150px 0;
+    }
+
+    /* about */
+    .skills {
+        flex-direction: column;
+        display: flex;
+    }
+
+    /* projects */
+    .projects-section .container {
+        display: flex;
+        flex-direction: column;
+    }
+
+    .projects {
+        display: block;
+    }
+
+
+    /* contact-bar-item */
+    .contact-bg {
```

Comment:
I think adding `max-width: 100%` in here would be good for mobile.  There's a small overflow that occurs because of this SVG on smaller screens:

<img width="378" alt="Screen Shot 2020-04-18 at 9 47 30 PM" src="https://user-images.githubusercontent.com/10561050/79666783-7d0d1480-81be-11ea-951d-d9abd6552a88.png">